src/blueapi/startup/example_devices.py

```diff
@@ -21,6 +23,11 @@ def theta(name="theta") -> SynAxisWithMotionEvents:
     )
 
 
+@device_factory(mock=True)
+def ppump() -> WatsonMarlow323Pump:
```

Comment:
Should: For internal blueapi tests/examples I would rather you make up your own device than depend on one from dodal, the one from dodal could be modified without our knowledge which makes the tests less stable. However I'm on the fence about this opinion so I'll also ask @DiamondJoseph